tests/test_enterprise/api/test_views.py

```diff
@@ -220,33 +185,28 @@ def test_enterprise_learner_entitlements(
                 (this includes enforcing data sharing consent at login and at enrollment) and enterprise learner
                  does not consent to share data.
             2. Full list of entitlements for all other cases.
-        Arguments:
-            enable_data_sharing_consent (bool): True if enterprise customer enables data sharing consent,
-                False it does not.
-            enforce_data_sharing_consent (str): string for the location at which enterprise customer enforces
-                data sharing consent, possible values are 'at_enrollment' and 'externally_managed'.
-            learner_consent_state (str): string containing the state of learner consent on data sharing,
-                possible values are 'not_set', 'enabled' and 'disabled'.
-            entitlements (list): A list of integers pointing to voucher ids generated in E-Commerce CAT tool.
-            expected_json (dict): A dict with structure and values corresponding to
-                the expected json from API endpoint.
```

Comment:
I didn't think it was useful to maintain an argument list doc for a test function. The names should speak for themselves.